### PR TITLE
Update CSS for data tables

### PR DESCRIPTION
**Pull request by Megan-Wright**
# Changes

For: https://github.com/tektoncd/dashboard/issues/837 and https://github.com/tektoncd/dashboard/issues/840

Fixes status icon being positioned above its status in PipelineRun and TaskRun tables, and extends the table header CSS to include the overflow column.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/components/Definitions/Definitions.scss [hunk 1]
```diff
@@ -40,6 +40,7 @@ main {
   .status-icon {
     margin-right: 0.5em;
     vertical-align: sub;
+    float: left;
```
Comment (steveodonovan) [inline src/components/Definitions/Definitions.scss]: I think this change should be added to pipelineRuns.scss under the packages dir, this causes the story book and third party consumes to still have the issue 

storybook screen grab with the changeset

Screenshot 2019-12-13 at 14 58 26

Comment (Megan-Wright) [reply src/components/Definitions/Definitions.scss]: ah ok, I'll update that now

File: packages/components/src/components/PipelineRuns/PipelineRuns.scss [hunk 2]
```diff
@@ -17,6 +17,7 @@ limitations under the License.
   .status-icon {
     margin-right: 0.5em;
     vertical-align: sub;
+    float: left;
```
Comment (carlos-logro) [inline packages/components/src/components/PipelineRuns/PipelineRuns.scss]: I mentioned in issue 840 that this css is not needed if the the div wrapper is removed from the icon. This is covered in PR #835 https://github.com/tektoncd/dashboard/blob/0c73c4086cbf9f54ede001c4fa122007a0caac1e/packages/components/src/components/PipelineRuns/PipelineRuns.js#L136

Comment (AlanGreene) [reply packages/components/src/components/PipelineRuns/PipelineRuns.scss]: @carlos-logro see my review comment, using flexbox on the container gives us a cleaner layout. If we could combine the 2 approaches (removing the container, and applying flexbox on a higher level container) that would be great.

Comment (Megan-Wright) [reply packages/components/src/components/PipelineRuns/PipelineRuns.scss]: @AlanGreene I made the change you suggested - this fixes everything nicely for the PipelineRun view but messes up the TaskRun view to look like this:
Screenshot 2019-12-16 at 14 03 53
Not sure why this is happening.... suggestions?

Comment (AlanGreene) [reply packages/components/src/components/PipelineRuns/PipelineRuns.scss]: You'll need to stop the icon from shrinking when other content in the flex container tries to steal space:

```
.definition .status {
  display: flex;

  .status-icon {
    flex-shrink: 0;
  }
}
```

Comment (Megan-Wright) [reply packages/components/src/components/PipelineRuns/PipelineRuns.scss]: thanks, made that change now :)

**Pull request merged**